pandas/tests/windows/test_rolling.py

```diff
@@ -0,0 +1,824 @@
+import pytest
+
+from datetime import datetime, timedelta
+from numpy.random import randn
+import numpy as np
+
+import pandas as pd
+from pandas import (Series, DataFrame, bdate_range,
+                    Timestamp, Index)
+import pandas.core.window as rwindow
+from pandas.errors import UnsupportedFunctionCall
+import pandas.util.testing as tm
+import pandas.util._test_decorators as td
+from pandas.compat import range
+
+N, K = 100, 10
+
+
+def assert_equal(left, right):
```

Comment:
don't repeat things. 2 ways to solve this, have a common.py and import, or you can use fixtures to do this properly.

Reply:
In this case I just need to insert `@pytest.fixture`?